2015-10-26 17:12:41,288 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.FollowUserIndexTask of type Tunynet.Tasks.Quartz.QuartzTask
Autofac.Core.DependencyResolutionException: An exception was thrown while executing a resolve operation. See the InnerException for details. ---> Lock obtain timed out: NativeFSLock@D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\FollowUser\write.lock: System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\FollowUser\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain() (See inner exception for details.) ---> Lucene.Net.Store.LockObtainFailedException: Lock obtain timed out: NativeFSLock@D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\FollowUser\write.lock: System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\FollowUser\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain() ---> System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\FollowUser\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Lucene.Net.Store.Lock.Obtain(Int64 lockWaitTimeout)
   在 Lucene.Net.Index.IndexWriter.Init(Directory d, Analyzer a, Boolean create, Boolean closeDir, IndexDeletionPolicy deletionPolicy, Boolean autoCommit, Int32 maxFieldLength, IndexingChain indexingChain, IndexCommit commit)
   在 Lucene.Net.Index.IndexWriter..ctor(Directory d, Analyzer a, Boolean create, MaxFieldLength mfl)
   在 Tunynet.Search.SearchEngine.InitSearcherManager()
   在 Tunynet.Search.SearchEngine.Initialize(String indexPath, Int32 mergeFactor, Analyzer analyzer)
   在 Tunynet.Search.SearchEngine..ctor(String indexPath)
   在 Spacebuilder.Search.SearcherFactory.GetSearchEngine(String indexPath) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\SearcherFactory.cs:行号 113
   在 Spacebuilder.Common.FollowUserSearcher..ctor(String name, String indexPath, Boolean asQuickSearch, Int32 displayOrder) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\User\Search\FollowUserSearcher.cs:行号 142
   在 Spacebuilder.Environments.Starter.<InitializeDIContainer>b__2a(IComponentContext c) 位置 d:\近乎_V4.2.0.0_深度源码版\Web\Starter.cs:行号 246
   在 Autofac.Builder.RegistrationBuilder.<>c__DisplayClass1`1.<ForDelegate>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Lifetime.LifetimeScope.GetOrCreateAndShare(Guid id, Func`1 creator)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   --- 内部异常堆栈跟踪的结尾 ---
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.TryResolveService(IComponentContext context, Service service, IEnumerable`1 parameters, Object& instance)
   在 Autofac.ResolutionExtensions.ResolveNamed[TService](IComponentContext context, String serviceName, IEnumerable`1 parameters)
   在 Tunynet.DIContainer.ResolveNamed[TService](String serviceName)
   在 Spacebuilder.Search.SearcherFactory.GetSearcher(String code) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\SearcherFactory.cs:行号 91
   在 Spacebuilder.Search.Tasks.FollowUserIndexTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\FollowUserIndexTask.cs:行号 41
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 17:12:41,288 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-6] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
Autofac.Core.DependencyResolutionException: An exception was thrown while executing a resolve operation. See the InnerException for details. ---> Lock obtain timed out: NativeFSLock@D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock: System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain() (See inner exception for details.) ---> Lucene.Net.Store.LockObtainFailedException: Lock obtain timed out: NativeFSLock@D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock: System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain() ---> System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Lucene.Net.Store.Lock.Obtain(Int64 lockWaitTimeout)
   在 Lucene.Net.Index.IndexWriter.Init(Directory d, Analyzer a, Boolean create, Boolean closeDir, IndexDeletionPolicy deletionPolicy, Boolean autoCommit, Int32 maxFieldLength, IndexingChain indexingChain, IndexCommit commit)
   在 Lucene.Net.Index.IndexWriter..ctor(Directory d, Analyzer a, Boolean create, MaxFieldLength mfl)
   在 Tunynet.Search.SearchEngine.InitSearcherManager()
   在 Tunynet.Search.SearchEngine.Initialize(String indexPath, Int32 mergeFactor, Analyzer analyzer)
   在 Tunynet.Search.SearchEngine..ctor(String indexPath)
   在 Spacebuilder.Search.SearcherFactory.GetSearchEngine(String indexPath) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\SearcherFactory.cs:行号 113
   在 Spacebuilder.Photo.PhotoSearcher..ctor(String name, String indexPath, Boolean asQuickSearch, Int32 displayOrder) 位置 d:\近乎_V4.2.0.0_深度源码版\Web\Applications\Photo\Search\PhotoSearcher.cs:行号 56
   在 Spacebuilder.Photo.PhotoConfig.<Initialize>b__0(IComponentContext c) 位置 d:\近乎_V4.2.0.0_深度源码版\Web\Applications\Photo\PhotoConfig.cs:行号 91
   在 Autofac.Builder.RegistrationBuilder.<>c__DisplayClass1`1.<ForDelegate>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Lifetime.LifetimeScope.GetOrCreateAndShare(Guid id, Func`1 creator)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   在 System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   在 System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   在 Autofac.Features.Collections.CollectionRegistrationSource.<>c__DisplayClass4.<RegistrationsFor>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   --- 内部异常堆栈跟踪的结尾 ---
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.ResolveService(IComponentContext context, Service service, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.Resolve[TService](IComponentContext context, IEnumerable`1 parameters)
   在 Tunynet.DIContainer.Resolve[TService]()
   在 Spacebuilder.Search.SearcherFactory.GetSearchersOfBaseLucene() 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\SearcherFactory.cs:行号 80
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 37
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 17:12:41,288 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-7] - Exception while running job DEFAULT.LuceneIndexOptimizeTask of type Tunynet.Tasks.Quartz.QuartzTask
Autofac.Core.DependencyResolutionException: An exception was thrown while executing a resolve operation. See the InnerException for details. ---> Lock obtain timed out: NativeFSLock@D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock: System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain() (See inner exception for details.) ---> Lucene.Net.Store.LockObtainFailedException: Lock obtain timed out: NativeFSLock@D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock: System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain() ---> System.IO.IOException: 文件“D:\近乎_V4.2.0.0_深度源码版\Web\App_Data\IndexFiles\Photo\write.lock”正由另一进程使用，因此该进程无法访问此文件。
   在 System.IO.__Error.WinIOError(Int32 errorCode, String maybeFullPath)
   在 System.IO.FileStream.Init(String path, FileMode mode, FileAccess access, Int32 rights, Boolean useRights, FileShare share, Int32 bufferSize, FileOptions options, SECURITY_ATTRIBUTES secAttrs, String msgPath, Boolean bFromProxy, Boolean useLongPath, Boolean checkHost)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access, FileShare share, Int32 bufferSize, FileOptions options, String msgPath, Boolean bFromProxy)
   在 System.IO.FileStream..ctor(String path, FileMode mode, FileAccess access)
   在 Lucene.Net.Store.NativeFSLock.Obtain()
   --- 内部异常堆栈跟踪的结尾 ---
   在 Lucene.Net.Store.Lock.Obtain(Int64 lockWaitTimeout)
   在 Lucene.Net.Index.IndexWriter.Init(Directory d, Analyzer a, Boolean create, Boolean closeDir, IndexDeletionPolicy deletionPolicy, Boolean autoCommit, Int32 maxFieldLength, IndexingChain indexingChain, IndexCommit commit)
   在 Lucene.Net.Index.IndexWriter..ctor(Directory d, Analyzer a, Boolean create, MaxFieldLength mfl)
   在 Tunynet.Search.SearchEngine.InitSearcherManager()
   在 Tunynet.Search.SearchEngine.Initialize(String indexPath, Int32 mergeFactor, Analyzer analyzer)
   在 Tunynet.Search.SearchEngine..ctor(String indexPath)
   在 Spacebuilder.Search.SearcherFactory.GetSearchEngine(String indexPath) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\SearcherFactory.cs:行号 113
   在 Spacebuilder.Photo.PhotoSearcher..ctor(String name, String indexPath, Boolean asQuickSearch, Int32 displayOrder) 位置 d:\近乎_V4.2.0.0_深度源码版\Web\Applications\Photo\Search\PhotoSearcher.cs:行号 56
   在 Spacebuilder.Photo.PhotoConfig.<Initialize>b__0(IComponentContext c) 位置 d:\近乎_V4.2.0.0_深度源码版\Web\Applications\Photo\PhotoConfig.cs:行号 91
   在 Autofac.Builder.RegistrationBuilder.<>c__DisplayClass1`1.<ForDelegate>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Lifetime.LifetimeScope.GetOrCreateAndShare(Guid id, Func`1 creator)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   在 System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   在 System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   在 Autofac.Features.Collections.CollectionRegistrationSource.<>c__DisplayClass4.<RegistrationsFor>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   --- 内部异常堆栈跟踪的结尾 ---
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.ResolveService(IComponentContext context, Service service, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.Resolve[TService](IComponentContext context, IEnumerable`1 parameters)
   在 Tunynet.DIContainer.Resolve[TService]()
   在 Spacebuilder.Search.SearcherFactory.GetSearchersOfBaseLucene() 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\SearcherFactory.cs:行号 80
   在 Spacebuilder.Search.Tasks.LuceneIndexOptimizeTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexOptimizeTask.cs:行号 41
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 17:16:04,228 [WARN] [tunynet] [28] - d:\近乎_V4.2.0.0_深度源码版\Web\Plugins\MediaParsers\Video\Youku.cshtml(10): error CS0103: 当前上下文中不存在名称“HttpWebHelper”--异常URL：http://127.0.0.1:8088/channel/parsemedia.aspx?mediatype=Video&url=http%3A%2F%2Fv.youku.com%2Fv_show%2Fid_XMTM1NjI5NDU0OA%3D%3D.html%3Ffrom%3Dy1.3-idx-uhome-1519-20887.205805-205902.9-1&_=1445850962939
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/43.0.2357.124 Safari/537.36

2015-10-26 17:50:30,474 [WARN] [tunynet] [50] - 值不能为 null。
参数名: input--异常URL：http://localhost:8088/channel/parsemedia.aspx?mediatype=Video&url=http%3A%2F%2Fv.youku.com%2Fv_show%2Fid_XMTM2MjY3ODg0MA%3D%3D.html%3Ff%3D26158504%26from%3Dy1.3-idx-uhome-1519-20887.205908-205909-205916.4-3&_=1445852955420
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 6.1; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/42.0.2311.152 Safari/537.36

2015-10-26 17:53:12,147 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-9] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 17:54:02,892 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-2] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 17:56:07,000 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 17:58:34,372 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:00:00,004 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-1] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:02:00,001 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-10] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:04:12,314 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-4] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:06:02,837 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-4] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:08:01,522 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:10:01,769 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 18:12:03,206 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
System.NullReferenceException: 未将对象引用设置到对象的实例。
   在 Tunynet.Search.SearchEngine.Commit()
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 43
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
2015-10-26 19:26:11,752 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-6] - Exception while running job DEFAULT.UpdateStageCountTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 已成功与服务器建立连接，但是在登录前的握手期间发生错误。 (provider: SSL Provider, error: 0 - 指定的句柄无效) ---> System.ComponentModel.Win32Exception (0x80004005): 指定的句柄无效
   在 System.Data.SqlClient.SqlInternalConnection.OnError(SqlException exception, Boolean breakConnection, Action`1 wrapCloseInAction)
   在 System.Data.SqlClient.TdsParser.ThrowExceptionAndWarning(TdsParserStateObject stateObj, Boolean callerHasConnectionLock, Boolean asyncClose)
   在 System.Data.SqlClient.TdsParser.ConsumePreLoginHandshake(Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity, Boolean& marsCapable)
   在 System.Data.SqlClient.TdsParser.Connect(ServerInfo serverInfo, SqlInternalConnectionTds connHandler, Boolean ignoreSniOpenTimeout, Int64 timerExpire, Boolean encrypt, Boolean trustServerCert, Boolean integratedSecurity, Boolean withFailover)
   在 System.Data.SqlClient.SqlInternalConnectionTds.AttemptOneLogin(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean ignoreSniOpenTimeout, TimeoutTimer timeout, Boolean withFailover)
   在 System.Data.SqlClient.SqlInternalConnectionTds.LoginNoFailover(ServerInfo serverInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString connectionOptions, SqlCredential credential, TimeoutTimer timeout)
   在 System.Data.SqlClient.SqlInternalConnectionTds.OpenLoginEnlist(TimeoutTimer timeout, SqlConnectionString connectionOptions, SqlCredential credential, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance)
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection()
   在 Tunynet.Common.CountRepository.UpdateStageCountPerDay(String tenantTypeId, String countType, Dictionary`2 countType2Days)
   在 Tunynet.Common.UpdateStageCountTask.Execute(TaskDetail taskDetail)
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context)
ClientConnectionId:d65d54e4-2514-4e06-a2ab-52139cb80afd
2016-04-11 10:28:45,811 [WARN] [tunynet] [13] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:28:47,794 [WARN] [tunynet] [10] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:28:48,331 [WARN] [tunynet] [31] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:28:48,516 [WARN] [tunynet] [8] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:28:48,722 [WARN] [tunynet] [56] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:28:48,915 [WARN] [tunynet] [13] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:30:07,275 [WARN] [tunynet] [7] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:33:09,454 [WARN] [tunynet] [34] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:33:18,647 [WARN] [tunynet] [16] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-04-11 10:36:34,668 [WARN] [tunynet] [31] - The ViewName Start Of Setup could not be found!--异常URL：http://127.0.0.1:1024/Install.aspx
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/48.0.2564.116 Safari/537.36

2016-05-03 10:59:46,166 [WARN] [tunynet] [7] - 给定关键字不在字典中。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 10:59:54,257 [WARN] [tunynet] [7] - 给定关键字不在字典中。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 11:05:36,812 [WARN] [tunynet] [32] - 给定关键字不在字典中。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 11:05:58,349 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-10] - Exception while running job DEFAULT.FollowUserIndexTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\GitHub\SNS\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.FetchFirstColumn(String sql, Object[] args) 位置 d:\GitHub\SNS\Infrastructure\PetaPoco\PetaPoco.Partial.cs:行号 97
   在 PetaPoco.Database.FetchFirstColumn(Sql sql) 位置 d:\GitHub\SNS\Infrastructure\PetaPoco\PetaPoco.Partial.cs:行号 88
   在 Tunynet.Common.Repositories.FollowRepository.GetRecentFollowerUserIds(DateTime lastStart) 位置 d:\GitHub\SNS\BusinessComponents\User\Follow\Repositories\FollowRepository.cs:行号 1044
   在 Tunynet.Common.FollowService.GetRecentFollowerUserIds(DateTime lastStart) 位置 d:\GitHub\SNS\BusinessComponents\User\Follow\FollowService.cs:行号 346
   在 Spacebuilder.Search.Tasks.FollowUserIndexTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\Common\Search\Tasks\FollowUserIndexTask.cs:行号 52
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\GitHub\SNS\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 11:06:28,393 [WARN] [tunynet] [39] - The ViewName Index Of the controllerName Index could not be found!--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 11:10:20,123 [WARN] [tunynet] [17] - The ViewName Index Of the controllerName Index could not be found!--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 11:11:55,724 [WARN] [tunynet] [21] - The ViewName Index Of the controllerName Index could not be found!--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:31:39,131 [WARN] [tunynet] [118] - 路由表中没有与提供的值匹配的路由。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:31:44,998 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-3] - Exception while running job DEFAULT.DeleteTrashDataTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Threading.ThreadAbortException: 正在中止线程。
   在 SNIReadSyncOverAsync(SNI_ConnWrapper* , SNI_Packet** , Int32 )
   在 SNINativeMethodWrapper.SNIReadSyncOverAsync(SafeHandle pConn, IntPtr& packet, Int32 timeout)
   在 System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()
   在 System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()
   在 System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()
   在 System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlCommand.RunExecuteNonQueryTds(String methodName, Boolean async, Int32 timeout, Boolean asyncWrite)
   在 System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite)
   在 System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   在 PetaPoco.Database.Execute(IEnumerable`1 sqls) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.Partial.cs:行号 78
   在 Tunynet.Common.CountRepository.DeleteTrashCount() 位置 d:\GitHub\SNS\BusinessComponents\Interaction\Count\Repositories\CountRepository.cs:行号 472
   在 Tunynet.Common.DeleteTrashDataTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Common\Tasks\DeleteTrashDataTask.cs:行号 34
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
2016-05-03 13:31:44,999 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-5] - Exception while running job DEFAULT.UpdateStageCountTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Threading.ThreadAbortException: 正在中止线程。
   在 SNIReadSyncOverAsync(SNI_ConnWrapper* , SNI_Packet** , Int32 )
   在 SNINativeMethodWrapper.SNIReadSyncOverAsync(SafeHandle pConn, IntPtr& packet, Int32 timeout)
   在 System.Data.SqlClient.TdsParserStateObject.ReadSniSyncOverAsync()
   在 System.Data.SqlClient.TdsParserStateObject.TryReadNetworkPacket()
   在 System.Data.SqlClient.TdsParserStateObject.TryPrepareBuffer()
   在 System.Data.SqlClient.TdsParserStateObject.TryReadByte(Byte& value)
   在 System.Data.SqlClient.TdsParser.TryRun(RunBehavior runBehavior, SqlCommand cmdHandler, SqlDataReader dataStream, BulkCopySimpleResultSet bulkCopyHandler, TdsParserStateObject stateObj, Boolean& dataReady)
   在 System.Data.SqlClient.SqlCommand.RunExecuteNonQueryTds(String methodName, Boolean async, Int32 timeout, Boolean asyncWrite)
   在 System.Data.SqlClient.SqlCommand.InternalExecuteNonQuery(TaskCompletionSource`1 completion, String methodName, Boolean sendToPipe, Int32 timeout, Boolean asyncWrite)
   在 System.Data.SqlClient.SqlCommand.ExecuteNonQuery()
   在 PetaPoco.Database.Execute(String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 558
   在 Tunynet.Common.CountRepository.UpdateStageCountPerDay(String tenantTypeId, String countType, Dictionary`2 countType2Days) 位置 d:\GitHub\SNS\BusinessComponents\Interaction\Count\Repositories\CountRepository.cs:行号 424
   在 Tunynet.Common.UpdateStageCountTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Interaction\Count\Tasks\UpdateStageCountTask.cs:行号 40
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
2016-05-03 13:35:51,960 [WARN] [tunynet] [118] - 执行处理程序“System.Web.Mvc.HttpHandlerUtil+ServerExecuteHttpHandlerAsyncWrapper”的子请求时出错。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:36:22,100 [WARN] [tunynet] [132] - 执行处理程序“System.Web.Mvc.HttpHandlerUtil+ServerExecuteHttpHandlerAsyncWrapper”的子请求时出错。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:39:17,985 [WARN] [tunynet] [97] - 路由表中没有与提供的值匹配的路由。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:42:27,860 [WARN] [tunynet] [99] - 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接)--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-6] - Exception while running job DEFAULT.UpdateStageCountTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 Tunynet.Common.CountRepository.UpdateStageCountPerDay(String tenantTypeId, String countType, Dictionary`2 countType2Days) 位置 d:\GitHub\SNS\BusinessComponents\Interaction\Count\Repositories\CountRepository.cs:行号 399
   在 Tunynet.Common.UpdateStageCountTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Interaction\Count\Tasks\UpdateStageCountTask.cs:行号 40
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-9] - Exception while running job DEFAULT.DeleteTrashDataTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Fetch[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 643
   在 PetaPoco.Database.Fetch[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 654
   在 Tunynet.Common.Repositories.TenantTypeRepository.Gets(String serviceKey, Nullable`1 applicationId) 位置 d:\GitHub\SNS\BusinessComponents\Common\TenantType\Repositories\TenantTypeRepository.cs:行号 57
   在 Tunynet.Common.TenantTypeService.Gets(String serviceKey, Nullable`1 applicationId) 位置 d:\GitHub\SNS\BusinessComponents\Common\TenantType\TenantTypeService.cs:行号 46
   在 Tunynet.Common.Repositories.FavoriteRepository.DeleteTrashDatas(String serviceKey) 位置 d:\GitHub\SNS\BusinessComponents\Common\Favorite\Repositories\FavoriteRepository.cs:行号 425
   在 Tunynet.Common.DeleteTrashDataTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Common\Tasks\DeleteTrashDataTask.cs:行号 28
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-7] - Exception while running job DEFAULT.LuceneIndexCommitTask of type Tunynet.Tasks.Quartz.QuartzTask
Autofac.Core.DependencyResolutionException: An exception was thrown while executing a resolve operation. See the InnerException for details. ---> 超时时间已到。超时时间已到，但是尚未从池中获取连接。出现这种情况可能是因为所有池连接均在使用，并且达到了最大池大小。 (See inner exception for details.) ---> System.InvalidOperationException: 超时时间已到。超时时间已到，但是尚未从池中获取连接。出现这种情况可能是因为所有池连接均在使用，并且达到了最大池大小。
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Fetch[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 643
   在 PetaPoco.Database.Fetch[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 654
   在 Tunynet.Common.Repositories.ApplicationDataRepository.GetAll(Int32 applicationId, String tenantTypeId) 位置 d:\GitHub\SNS\BusinessComponents\Common\ApplicationData\Repositories\ApplicationDataRepository.cs:行号 264
   在 Tunynet.Common.ApplicationDataService.GetLong(Int32 applicationId, String dataKey, String tenantTypeId) 位置 d:\GitHub\SNS\BusinessComponents\Common\ApplicationData\ApplicationDataService.cs:行号 145
   在 Tunynet.Common.AuditService.GetPubliclyAuditStatus(Int32 applicationId) 位置 d:\GitHub\SNS\BusinessComponents\Regulate\Audit\AuditService.cs:行号 107
   在 Spacebuilder.Ask.AskSearcher..ctor(String name, String indexPath, Boolean asQuickSearch, Int32 displayOrder) 位置 d:\GitHub\SNS\Web\Applications\Ask\Search\AskSearcher.cs:行号 49
   在 Spacebuilder.Ask.AskConfig.<Initialize>b__1(IComponentContext c) 位置 d:\GitHub\SNS\Web\Applications\Ask\AskConfig.cs:行号 114
   在 Autofac.Builder.RegistrationBuilder.<>c__DisplayClass1`1.<ForDelegate>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Lifetime.LifetimeScope.GetOrCreateAndShare(Guid id, Func`1 creator)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   在 System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   在 System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   在 Autofac.Features.Collections.CollectionRegistrationSource.<>c__DisplayClass4.<RegistrationsFor>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   --- 内部异常堆栈跟踪的结尾 ---
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.TryResolveService(IComponentContext context, Service service, IEnumerable`1 parameters, Object& instance)
   在 Autofac.ResolutionExtensions.ResolveService(IComponentContext context, Service service, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.Resolve[TService](IComponentContext context, IEnumerable`1 parameters)
   在 Tunynet.DIContainer.Resolve[TService]() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Environments\DIContainer.cs:行号 43
   在 Spacebuilder.Search.SearcherFactory.GetSearchersOfBaseLucene() 位置 d:\GitHub\SNS\Common\Search\SearcherFactory.cs:行号 80
   在 Spacebuilder.Search.Tasks.LuceneIndexCommitTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\Common\Search\Tasks\LuceneIndexCommitTask.cs:行号 37
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-1] - Exception while running job DEFAULT.LuceneIndexOptimizeTask of type Tunynet.Tasks.Quartz.QuartzTask
Autofac.Core.DependencyResolutionException: An exception was thrown while executing a resolve operation. See the InnerException for details. ---> 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) (See inner exception for details.) ---> System.Data.SqlClient.SqlException: 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception: 找不到网络路径。
   --- 内部异常堆栈跟踪的结尾 ---
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Fetch[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 643
   在 PetaPoco.Database.Fetch[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 654
   在 Tunynet.Common.Repositories.ApplicationDataRepository.GetAll(Int32 applicationId, String tenantTypeId) 位置 d:\GitHub\SNS\BusinessComponents\Common\ApplicationData\Repositories\ApplicationDataRepository.cs:行号 264
   在 Tunynet.Common.ApplicationDataService.GetLong(Int32 applicationId, String dataKey, String tenantTypeId) 位置 d:\GitHub\SNS\BusinessComponents\Common\ApplicationData\ApplicationDataService.cs:行号 145
   在 Tunynet.Common.AuditService.GetPubliclyAuditStatus(Int32 applicationId) 位置 d:\GitHub\SNS\BusinessComponents\Regulate\Audit\AuditService.cs:行号 107
   在 Spacebuilder.Ask.AskSearcher..ctor(String name, String indexPath, Boolean asQuickSearch, Int32 displayOrder) 位置 d:\GitHub\SNS\Web\Applications\Ask\Search\AskSearcher.cs:行号 49
   在 Spacebuilder.Ask.AskConfig.<Initialize>b__1(IComponentContext c) 位置 d:\GitHub\SNS\Web\Applications\Ask\AskConfig.cs:行号 114
   在 Autofac.Builder.RegistrationBuilder.<>c__DisplayClass1`1.<ForDelegate>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Lifetime.LifetimeScope.GetOrCreateAndShare(Guid id, Func`1 creator)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 System.Linq.Enumerable.WhereSelectArrayIterator`2.MoveNext()
   在 System.Linq.Buffer`1..ctor(IEnumerable`1 source)
   在 System.Linq.Enumerable.ToArray[TSource](IEnumerable`1 source)
   在 Autofac.Features.Collections.CollectionRegistrationSource.<>c__DisplayClass4.<RegistrationsFor>b__0(IComponentContext c, IEnumerable`1 p)
   在 Autofac.Core.Activators.Delegate.DelegateActivator.ActivateInstance(IComponentContext context, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Activate(IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.InstanceLookup.Execute()
   在 Autofac.Core.Resolving.ResolveOperation.GetOrCreateInstance(ISharingLifetimeScope currentOperationScope, IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   --- 内部异常堆栈跟踪的结尾 ---
   在 Autofac.Core.Resolving.ResolveOperation.Execute(IComponentRegistration registration, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.TryResolveService(IComponentContext context, Service service, IEnumerable`1 parameters, Object& instance)
   在 Autofac.ResolutionExtensions.ResolveService(IComponentContext context, Service service, IEnumerable`1 parameters)
   在 Autofac.ResolutionExtensions.Resolve[TService](IComponentContext context, IEnumerable`1 parameters)
   在 Tunynet.DIContainer.Resolve[TService]() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Environments\DIContainer.cs:行号 43
   在 Spacebuilder.Search.SearcherFactory.GetSearchersOfBaseLucene() 位置 d:\GitHub\SNS\Common\Search\SearcherFactory.cs:行号 80
   在 Spacebuilder.Search.Tasks.LuceneIndexOptimizeTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\Common\Search\Tasks\LuceneIndexOptimizeTask.cs:行号 41
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-8] - Exception while running job DEFAULT.OnlineUserTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Query[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 983
   在 PetaPoco.Database.FirstOrDefault[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 1143
   在 Tunynet.Common.Repositories.SettingsRepository`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\Repositories\SettingsRepository.cs:行号 57
   在 Tunynet.Common.SettingManager`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\SettingsManager.cs:行号 22
   在 Tunynet.Common.Repositories.OnlineUserRepository.Refresh(ConcurrentDictionary`2 OnlineUsersForProcess) 位置 d:\GitHub\SNS\BusinessComponents\User\OnlineUser\Repositories\OnlineUserRepository.cs:行号 143
   在 Tunynet.Common.OnlineUserService.Refresh() 位置 d:\GitHub\SNS\BusinessComponents\User\OnlineUser\OnlineUserService.cs:行号 144
   在 Tunynet.Common.Tasks.OnlineUserTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\User\OnlineUser\Tasks\OnlineUserTask.cs:行号 30
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-2] - Exception while running job DEFAULT.EmailTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Query[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 983
   在 PetaPoco.Database.FirstOrDefault[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 1143
   在 Tunynet.Common.Repositories.SettingsRepository`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\Repositories\SettingsRepository.cs:行号 57
   在 Spacebuilder.Common.EmailSettingsManager.Tunynet.Email.IEmailSettingsManager.Get() 位置 d:\GitHub\SNS\Common\Email\EmailSettingsManager.cs:行号 37
   在 Tunynet.Email.Tasks.EmailTask.Execute(TaskDetail taskDetail) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Email\Tasks\EmailQueueTask.cs:行号 40
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-5] - Exception while running job DEFAULT.DeleteTrashTemporaryAttachmentsTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Query[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 983
   在 PetaPoco.Database.FirstOrDefault[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 1143
   在 Tunynet.Common.Repositories.SettingsRepository`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\Repositories\SettingsRepository.cs:行号 57
   在 Tunynet.Common.SettingManager`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\SettingsManager.cs:行号 22
   在 Tunynet.Common.AttachmentService`1.DeleteTrashTemporaryAttachments() 位置 d:\GitHub\SNS\BusinessComponents\Common\Attachments\AttachmentService`1.cs:行号 510
   在 Tunynet.Common.DeleteTrashTemporaryAttachmentsTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Common\Attachments\Tasks\DeleteTrashTemporaryAttachmentsTask.cs:行号 30
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:42:27,860 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-10] - Exception while running job DEFAULT.SendReminderTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Query[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 983
   在 PetaPoco.Database.FirstOrDefault[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 1143
   在 Tunynet.Common.Repositories.SettingsRepository`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\Repositories\SettingsRepository.cs:行号 57
   在 Tunynet.Common.SettingManager`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\SettingsManager.cs:行号 22
   在 Tunynet.Common.SendReminderTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Communication\Reminder\Tasks\SendReminderTask.cs:行号 37
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:42:27,860 [WARN] [tunynet] [129] - 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接)--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:43:27,157 [WARN] [tunynet] [97] - 执行处理程序“System.Web.Mvc.HttpHandlerUtil+ServerExecuteHttpHandlerAsyncWrapper”的子请求时出错。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:44:38,984 [WARN] [tunynet] [128] - 执行处理程序“System.Web.Mvc.HttpHandlerUtil+ServerExecuteHttpHandlerAsyncWrapper”的子请求时出错。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 13:47:01,985 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-7] - Exception while running job DEFAULT.DeleteTrashTemporaryAttachmentsTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 PetaPoco.Database.Query[T](String sql, Object[] args) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 890
   在 PetaPoco.Database.Query[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 983
   在 PetaPoco.Database.FirstOrDefault[T](Sql sql) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\PetaPoco\PetaPoco.cs:行号 1143
   在 Tunynet.Common.Repositories.SettingsRepository`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\Repositories\SettingsRepository.cs:行号 57
   在 Tunynet.Common.SettingManager`1.Get() 位置 d:\GitHub\SNS\BusinessComponents\Common\Settings\SettingsManager.cs:行号 22
   在 Tunynet.Common.AttachmentService`1.DeleteTrashTemporaryAttachments() 位置 d:\GitHub\SNS\BusinessComponents\Common\Attachments\AttachmentService`1.cs:行号 510
   在 Tunynet.Common.DeleteTrashTemporaryAttachmentsTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\Common\Attachments\Tasks\DeleteTrashTemporaryAttachmentsTask.cs:行号 30
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\近乎_V4.2.0.0_深度源码版\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 13:58:50,111 [WARN] [tunynet] [119] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:00:02,027 [WARN] [tunynet] [124] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:03:04,911 [WARN] [tunynet] [122] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:05:38,104 [WARN] [tunynet] [108] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:05:39,449 [WARN] [tunynet] [119] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:19:14,581 [WARN] [tunynet] [111] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:19:26,332 [WARN] [tunynet] [111] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:19:27,400 [WARN] [tunynet] [107] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:19:27,782 [WARN] [tunynet] [110] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:19:28,229 [WARN] [tunynet] [122] - 无法使用前导 .. 在顶级目录上退出。--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 14:21:39,739 [ERROR] [tunynet] [DefaultQuartzScheduler_Worker-4] - Exception while running job DEFAULT.ExecVisitQueueTask of type Tunynet.Tasks.Quartz.QuartzTask
System.Data.SqlClient.SqlException (0x80131904): 在与 SQL Server 建立连接时出现与网络相关的或特定于实例的错误。未找到或无法访问服务器。请验证实例名称是否正确并且 SQL Server 已配置为允许远程连接。 (provider: Named Pipes Provider, error: 40 - 无法打开到 SQL Server 的连接) ---> System.ComponentModel.Win32Exception (0x80004005): 找不到网络路径。
   在 System.Data.SqlClient.SqlInternalConnectionTds..ctor(DbConnectionPoolIdentity identity, SqlConnectionString connectionOptions, SqlCredential credential, Object providerInfo, String newPassword, SecureString newSecurePassword, Boolean redirectedUserInstance, SqlConnectionString userConnectionOptions, SessionData reconnectSessionData, DbConnectionPool pool, String accessToken, Boolean applyTransientFaultHandling)
   在 System.Data.SqlClient.SqlConnectionFactory.CreateConnection(DbConnectionOptions options, DbConnectionPoolKey poolKey, Object poolGroupProviderInfo, DbConnectionPool pool, DbConnection owningConnection, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionFactory.CreatePooledConnection(DbConnectionPool pool, DbConnection owningObject, DbConnectionOptions options, DbConnectionPoolKey poolKey, DbConnectionOptions userOptions)
   在 System.Data.ProviderBase.DbConnectionPool.CreateObject(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.UserCreateRequest(DbConnection owningObject, DbConnectionOptions userOptions, DbConnectionInternal oldConnection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, UInt32 waitForMultipleObjectsTimeout, Boolean allowCreate, Boolean onlyOneCheckConnection, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionPool.TryGetConnection(DbConnection owningObject, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionFactory.TryGetConnection(DbConnection owningConnection, TaskCompletionSource`1 retry, DbConnectionOptions userOptions, DbConnectionInternal oldConnection, DbConnectionInternal& connection)
   在 System.Data.ProviderBase.DbConnectionInternal.TryOpenConnectionInternal(DbConnection outerConnection, DbConnectionFactory connectionFactory, TaskCompletionSource`1 retry, DbConnectionOptions userOptions)
   在 System.Data.SqlClient.SqlConnection.TryOpenInner(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.TryOpen(TaskCompletionSource`1 retry)
   在 System.Data.SqlClient.SqlConnection.Open()
   在 PetaPoco.Database.OpenSharedConnection() 位置 d:\GitHub\SNS\Infrastructure\PetaPoco\PetaPoco.cs:行号 191
   在 Tunynet.Common.Repositories.VisitRepository.ExecQueue() 位置 d:\GitHub\SNS\BusinessComponents\User\Visit\Repositories\VisitRepository.cs:行号 152
   在 Tunynet.Common.ExecVisitQueueTask.Execute(TaskDetail taskDetail) 位置 d:\GitHub\SNS\BusinessComponents\User\Visit\Tasks\ExecVisitQueueTask.cs:行号 29
   在 Tunynet.Tasks.Quartz.QuartzTask.Execute(IJobExecutionContext context) 位置 d:\GitHub\SNS\Infrastructure\Tasks\Quartz\QuartzTask.cs:行号 48
ClientConnectionId:00000000-0000-0000-0000-000000000000
Error Number:53,State:0,Class:20
2016-05-03 16:48:04,337 [ERROR] [tunynet] [137] - 安全校验过滤器,校验CSRF Token时失败。
System.Web.Mvc.HttpAntiForgeryException (0x80004005): 所需的防伪 Cookie“__RequestVerificationToken”不存在。
   在 System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   在 System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   在 Spacebuilder.Common.SecurityFilter.OnAuthorization(AuthorizationContext filterContext) 位置 d:\GitHub\SNS\Common\Controllers\Filters\SecurityFilter.cs:行号 50
2016-05-03 16:48:04,498 [ERROR] [tunynet] [137] - 安全校验过滤器,校验CSRF Token时失败。
System.Web.Mvc.HttpAntiForgeryException (0x80004005): 所需的防伪 Cookie“__RequestVerificationToken”不存在。
   在 System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   在 System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   在 Spacebuilder.Common.SecurityFilter.OnAuthorization(AuthorizationContext filterContext) 位置 d:\GitHub\SNS\Common\Controllers\Filters\SecurityFilter.cs:行号 50
2016-05-03 16:48:04,507 [WARN] [tunynet] [137] - 无法在发送 HTTP 标头之后进行重定向。--异常URL：http://127.0.0.1:1024/
Http传输方式：POST
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 16:48:05,161 [ERROR] [tunynet] [142] - 安全校验过滤器,校验CSRF Token时失败。
System.Web.Mvc.HttpAntiForgeryException (0x80004005): 所需的防伪 Cookie“__RequestVerificationToken”不存在。
   在 System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   在 System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   在 Spacebuilder.Common.SecurityFilter.OnAuthorization(AuthorizationContext filterContext) 位置 d:\GitHub\SNS\Common\Controllers\Filters\SecurityFilter.cs:行号 50
2016-05-03 16:48:05,163 [ERROR] [tunynet] [142] - 安全校验过滤器,校验CSRF Token时失败。
System.Web.Mvc.HttpAntiForgeryException (0x80004005): 所需的防伪 Cookie“__RequestVerificationToken”不存在。
   在 System.Web.Helpers.AntiXsrf.TokenValidator.ValidateTokens(HttpContextBase httpContext, IIdentity identity, AntiForgeryToken sessionToken, AntiForgeryToken fieldToken)
   在 System.Web.Helpers.AntiXsrf.AntiForgeryWorker.Validate(HttpContextBase httpContext)
   在 Spacebuilder.Common.SecurityFilter.OnAuthorization(AuthorizationContext filterContext) 位置 d:\GitHub\SNS\Common\Controllers\Filters\SecurityFilter.cs:行号 50
2016-05-03 16:48:05,163 [WARN] [tunynet] [142] - 无法在发送 HTTP 标头之后进行重定向。--异常URL：http://127.0.0.1:1024/
Http传输方式：POST
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400

2016-05-03 16:51:42,364 [WARN] [tunynet] [142] - <log4net.Error>Exception during StringFormat: 输入字符串的格式不正确。 <format>“9792”在代码块开头无效。只有标识符、关键字、注释、“(”和“{”才有效。
--异常URL：http://127.0.0.1:1024/
Http传输方式：GET
用户IP：127.0.0.1
用户代理信息：Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/47.0.2526.80 Safari/537.36 Core/1.47.163.400 QQBrowser/9.3.7175.400
</format><args>{System.Web.HttpParseException (0x80004005): “9792”在代码块开头无效。只有标识符、关键字、注释、“(”和“{”才有效。

   在 System.Web.WebPages.Razor.RazorBuildProvider.EnsureGeneratedCode()
   在 System.Web.WebPages.Razor.RazorBuildProvider.get_CodeCompilerType()
   在 System.Web.Compilation.BuildProvider.GetCompilerTypeFromBuildProvider(BuildProvider buildProvider)
   在 System.Web.Compilation.BuildProvidersCompiler.ProcessBuildProviders()
   在 System.Web.Compilation.BuildProvidersCompiler.PerformBuild()
   在 System.Web.Compilation.BuildManager.CompileWebFile(VirtualPath virtualPath)
   在 System.Web.Compilation.BuildManager.GetVPathBuildResultInternal(VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   在 System.Web.Compilation.BuildManager.GetVPathBuildResultWithNoAssert(HttpContext context, VirtualPath virtualPath, Boolean noBuild, Boolean allowCrossApp, Boolean allowBuildInPrecompile, Boolean throwIfNotFound, Boolean ensureIsUpToDate)
   在 System.Web.Compilation.BuildManager.GetVirtualPathObjectFactory(VirtualPath virtualPath, HttpContext context, Boolean allowCrossApp, Boolean throwIfNotFound)
   在 System.Web.Compilation.BuildManager.GetCompiledType(VirtualPath virtualPath)
   在 System.Web.Mvc.BuildManagerCompiledView.Render(ViewContext viewContext, TextWriter writer)
   在 System.Web.Mvc.ViewResultBase.ExecuteResult(ControllerContext context)
   在 System.Web.Mvc.ControllerActionInvoker.<>c__DisplayClass1a.<InvokeActionResultWithFilters>b__17()
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilter(IResultFilter filter, ResultExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilter(IResultFilter filter, ResultExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilter(IResultFilter filter, ResultExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilter(IResultFilter filter, ResultExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilter(IResultFilter filter, ResultExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.ControllerActionInvoker.InvokeActionResultFilter(IResultFilter filter, ResultExecutingContext preContext, Func`1 continuation)
   在 System.Web.Mvc.Async.AsyncControllerActionInvoker.<>c__DisplayClass25.<BeginInvokeAction>b__22(IAsyncResult asyncResult)}</args></log4net.Error>
